### ISSUE TITLE
1.9: Must explicitly set printWidth config for Markdown files

### ISSUE THREAD

**Issue by KSuttle-cng** [prettier/prettier#3410]
**Prettier 1.9.0**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEBiVACAKgSzAazhgwBscp8kAdKAfntppvQwGV4AHAZwwCc4AjgFcc-ACYYYESXC4wmUAFQYA2hgC6qgMIALOAQwQhxGDpw8ARrwCGUMDtIQw1kiQCe6gBQ6YMbkgB6AL0SDgA6AHMcUyELMMgAWwDrXhg8ElkA+318cgiAWiMYfI4hV3z+YVkYLnySJxd3AIBKGmU1TQAlISgMAB58CzEAPjcUqD6AwZHDXtNzDCtbe0lpcjlGjDE4DgRtuxxZNtUNBUwtaBheHAtjPLOWAEEMKDgAdwwAMzhrGCF+BQARjCGAABlFiNkCEUtnAAG5weocUE0YFgiEYUquDD5CpwCzWLhwFFQNHg6IYKH4GH5CwYAAajPpoIwbz0-AZTIwCzcRl4lJSEigQgSFjgvFRIIA6td4DI5FxJRgZdE4JSINslSq5WInCKEDBfjhoErHtiIKZxbC4ZJCfgePwSL84BIpBhef8MHIILw3EqAPoASQAYv73UYMDprAiMAlrNtY2U0hwMuqEglojwxP88uHPfVnCRrYiIBwEgbMRlCXAaBg6-wCUSMLYJFxhISHJaEtzYNJrF68qnEhniH1toacCQuMN+lwRXHfcMAAriz4+jNQCLNqC1uvkeA2MBpGMN6uTOfplJuYa7-ocG+kkHk4inpv5HAYAASAFFHgARAA-ABWFk2XFNUgO5HhLReEUxX5CBPjTEceA9WN4zgMIFAAQhUQM4wiNVEIwQNYHFawjxwGNOnxasvACHACMyV9MI4TdWh3KAZ1IyQ9GbCwIBjLgwH4BAuB0C1WTVFI1X3cjKIRdw+Forhc0g4dMy+XgIG7PlHELGgZ2+X5-jVJY7B0EEePmHhhIQFJjQAGik1lbBMaRABwCNshA7QBcAl42QIOQzT92kYFDIwYMfQCr5RDkUhyDVNC3knIt+BTCi1Rgt4fQkDyODwfB-JS0xtzy3h-Oijz+By3gxF8iLy1sXM3jVN43NWDBCB2XiFg04hbkhKNNyyvjyy4LhrEIkEov5GD+Djch7i4jAABZgpqFyUuxdKnTANV8sK4rogcDzPkq-kzpwAAPIQOH8rhpFMX4XO8jsAqJCL+p4FstnMZw6oC0RY1kSbpuVPRejQmStmgOBnMmmNfrga6KXmlSaxWl1omi36PX5FjWVSxY1V2Xg114csxCwziAE09OfRwIhZcSjBIV1IdmJTWY+fGfp+gdNyHHSRxwvCmMMJCAHEKQAGQgCJ6MYqbMnqCIwjYiION3SZ7xoSZx2sSdpwUABOEFHk+A91SgT5RA3Ld8flTaOEJLhnLIWBc1dibZGcmCOG0gArfQBpEdmfqgCR8dvfqQYmlWvggVwIDeHhICgBEvegZzoY5fh4zcTq5wsEc835JcyiLGiqjkGmyQxQmfRwKIoACbZFNLUD2TVDuSw4KCYoWi0zJsCzeJevPMbrDAIggXM3TFL1YhHNJN3LzEq+U2viCmo2oDr29lCwMweDE0zoKjXeoCL4e5XsWxCJ+jksQyCRQo34y-g5cyVnIW86wB20mTQwdUrTkFZGYFYloi6tQ5PubS2Z9rUyPhgemQgMD+jjG4MMvAeg9jdOWXghEbafDIEed2vUeALAgPgawbgQToIFL0O2UcYrWGMBJCUvQZ6SxihnMheBiAQJ3saXoFgi71FobmF6ME7aphKg4AR5DNoQMtAAjAMsYCfliIsbSadxQgi0JsLshgERzWkCEAeaF+CPRIDGNkeAHDmNUtAPh6ieEz0gAmVmZQJBLyEESFBj4IYICHhjSMhJl5gH2hNT4Vci6JBTEQF0zkTHVB4D7Shns15biyX7W8v0gEQBDkeRY4cxCUKnspQunVSjiSVIzepDh8gYObq3TkjISRogAEL6KbG6Z8OhYjxB0iyX6SSMhyidq-beQhqjNh4FAdcLgewBVvFYVORJuHMGVPiDAjwOApjwEaE0nET4LEDiU0OrkeCEVeDYeAEhFE0BUI8TcZQUgYC0LLQMXgfB+C4IEAIEJhlxESMkD5TpeCQoiJ83g+QwBkGaBgFxYiMDAgAAxhGxQ8DAf54T93LLAL04pzEKG6L0UE69tkIhZBTZs1pSW8HMSCAActGFuzpOqggBRwYFBYXASTkEgVaAAmTFmKAighBCfaSRyaDbSLBwqQcY0iFiUo6CA8ZuRITQg-Ea24i7ERgo9f4+1YoZC4DTZg5wNRqmEtYT4a52bLRoJSsE697nkXvjpDgcMSVJIDcUKA1hywsjdN6x50kXjvDTP614sBGERmcJxFwj0MCBLVNSrcUbuViFEKHKicAAgFV2AEWleAS1IrdgEBIGoyjEhtVAFgPSKkUvwTm8pk4xARukLcHtMUrmlJgLKviA72bNlSDgT4FFNpKpJl6KQ4g1keNBAWuQ0q-r8CPD6BhGAACqAy+KghKIg+lTotwMv7FcpBaQ3ETuCbajAlKoC5h6GjDJHaqXr3gHIPtGAUb6GMKNNUH6TAZNRTgawryADSKQ4z-N8P4IIdDKbWAqD0B5kQTojONM0ZtLBX25j2PkKQ+Q9jO0VJxD1Xa4BiuJJ1IDYAQMxVI+Ryjf7Npwmg68pc2krhzp9EhvwwK3jiY1gJw8y6uNhB9BEFoIIelwApmqPBUA32-r4lxngcZCDLwBDfCMMNK1aflQPHj-Yu20qbXi4M-xLT8isQoLA0hCLEHrRyKxswYrvLhdC75vzM1Ni7VYlk0U56Uj0AYGEHi3lQq+T8wML7fx-gALLfhEyhkFuHwU6VhfCgr0LEVkACBYeoFg62EgPAEToqWMthASGIAjNAQCORAKWe9B9kCgBSPopcMlusoGjPPMQbWQDVeQLOqc8MQBLAIEQVgrswB5GQFceZ7XdRgCm+m2b6xxQwH41NOMO2ZvtaDlwa6fSKKEBgKwMNcBZaJVO0SdrRRSgwDFS92brteDbOQCABc+BdRvCgONwO+4pQ4DEKYZAAAOTF7WrlEhlNYDgAPA6BRZXAcblQRD8COxEE7SBpuvZAESDMa28GzdUkLOAABFIQI8qcbZAIaCwUOYc6GQGK9rgnJx5AuJeAHsggLjazVgawFguDfYAL6y6AA)
```sh
# Options (if any):
{
  "singleQuote": true,
  "trailingComma": "es5"
}
```

**Input:**
See playground

**Output:**
```
[error] Invalid printWidth value.
[error] Expected an integer, but received: "off"
```

**Expected behavior:**
No printWidth config needed.

I do not get this error with the following config:
```diff
{
  "singleQuote": true,
  "trailingComma": "es5",
+  "printWidth": 80
}

**Comment by suchipi** [comment]
I'm not seeing this issue in the playground- could you try a force refresh? (Shift+F5, Cmd+Shift+R, etc)

**Comment by KSuttle-cng** [comment]
Right, because the playground has the printWidth setting applied.

**Comment by suchipi** [comment]
Ah, right. My bad

**Comment by KSuttle-cng** [comment]
No worries!

**Comment by duailibe** [comment]
Can't reproduce this... Could you share the exact command you are running and your (if any) config files?

**Comment by KSuttle-cng** [comment]
```json
"prettier --write '{./,src}/**/**/{*.ts,*.js,*.json,*.scss,*.css,*.md}'",
```

Config file is in the initial comment. Only 2 settings.

**Comment by duailibe** [comment]
Gotcha :+1:

Another question, do you have a .editorConfig file?

**Comment by KSuttle-cng** [comment]
Yes. It's this:

```editorConfig
root = true

[*]
charset = utf-8
indent_style = space
indent_size = 2
insert_final_newline = true
trim_trailing_whitespace = false

[*.md]
max_line_length = off
trim_trailing_whitespace = false
```

**Comment by duailibe** [comment]
Yeah, we're doing right `printWidth = max_line_length`, which is why the error message says it received "off".

**Comment by KSuttle-cng** [comment]
Ah that makes sense. Impressive sleuthing skills!

**Comment by KSuttle-cng** [comment]
Ooh yeah that is a tricky one: https://github.com/editorconfig/editorconfig/wiki/EditorConfig-Properties#max_line_length

**Comment by duailibe** [comment]
Hah thanks! I opened a PR on the dependency that handles this conversion and will bump the version on prettier after it lands. We'll probably be able to release a fix in 1.9.1

**Comment by KSuttle-cng** [comment]
Thank you!

**Comment by duailibe** [comment]
The fix was released in 1.9.1!
